src/main/java/com/google/ehub/servlets/ProfileServlet.java

```diff
@@ -38,7 +39,7 @@
 public class ProfileServlet extends HttpServlet {
   private static final String NAME_PROPERTY_KEY = "name";
   private static final String EMAIL_PARAMETER_KEY = "email";
-  private static final String USERNAME_PROPERTY_KEY = "username";
+  private static final String USERNAME_PARAMETER_KEY = "username";
```

Comment:
If you are changing PROPERTY -> PARAMETER for some constants, might as well do it for all. Otherwise the mismatch can be confusing

Reply:
OK I renamed all of them to parameter for consistency.